zip-1014.rst

```diff
@@ -61,19 +66,24 @@ Motivation
 
 Starting at Zcash's first halving in October 2020, by default 100% of the block
 subsidies will be allocated to miners, and no further funds will be automatically
-allocated to research, development, and outreach. Consequently, no substantial
-new funding may be available to existing teams dedicated to Zcash: the Electric
-Coin Company (ECC), the Zcash Foundation (ZF), and the many entities funded by
-the ZF grant program.
+allocated for charitable, educational, or scientific purposes, such as  research,
+development, and outreach. Consequently, no substantial
+new funding may be available to existing teams dedicated to furthering such charitable
+or educational purposes: the Electric Coin Company (ECC), the Zcash Foundation (ZF), and
+the many entities funded by the ZF grant program.
 
 There is a need to strike a balance between incentivizing the security of the
-consensus protocol (i.e., mining) versus other crucial aspects of the Zcash
-security and functionality, such as development and outreach.
+consensus protocol (i.e., mining) versus crucial charitable, educational, and/or scientific aspects, such as research, development and outreach.
 
 Furthermore, there is a need to balance the sustenance of ongoing work by the
 current teams dedicated to Zcash, with encouraging decentralization and growth
 of independent development teams.
 
+For these reasons, the Zcash Community desires to allocate and
+contribute a slice of the block subsidies otherwise allocated to
+miners as a donation to support charitable, educational, and
+scientific activities within the meaning of Section 501(c)(3) of the
+Code.
```

Comment:
```suggestion
scientific activities within the meaning of Section 501(c)(3).
```

Reply:
"of the Code" threw me off / confused me a bit, also, and I guessed it means US law. If it's going to be included, it helps (at least for readers like me) to say "the US Code". What is the chance any current or future reader won't be able to determine what Section 501(c)(3) refers to?

Reply:
This would be addressed by https://github.com/zcash/zips/pull/409/files#r508559299